examples/jquery-bootstrap/js/app.js

```diff
@@ -3,10 +3,10 @@
 var BootstrapButton = React.createClass({
   render: function() {
     return (
-      <a {...this.props}
+      <a onClick={this.props.onclick}
```

Comment:
Well onClick is meant to work for component but it doesnt i thought it prevents confusion maybe